### DISCUSSION TITLE
特定のキーが反応しない

### DISCUSSION THREAD

**Discussion by TAKAYUKISATAKE** [Yowkees/keyball#514] [Q&A] [answered]
添付画像のキーが反応したりしなかったりしています。対応方法がわからず行き詰まっております。
![IMG_2523](https://github.com/Yowkees/keyball/assets/164729456/1e2d1e42-6f7a-44a5-9c7a-ab6531ca3fa4)
下記やってみたことです。
キースイッチを動いているものと交換した➡︎反応せず
下記画像の様にショートさせてみた➡︎反応した
![IMG_2524](https://github.com/Yowkees/keyball/assets/164729456/0f93d94f-4444-4fb8-917b-290a741f20a5)

ご教授のほどよろしくお願いいたします。

**Comment by Ryo624** [comment, marked as answer]
ご連絡ありがとうございます。

キー単体での反応が無いということですので、キーソケットもしくはダイオードが原因となる可能性が高いです。
写真に赤い丸で示したダイオードが今回のキーに対応するダイオードですので、キーソケットのはんだ付けと併せてやり直していただければと思います。
![316294975-0f93d94f-4444-4fb8-917b-290a741f20a5](https://github.com/Yowkees/keyball/assets/143698349/2aa4f268-b72e-4564-86ef-906de7960154)

**Reply by TAKAYUKISATAKE** [reply]
ありがとうございます。
やってみます！
